### PR TITLE
troll: suffix for LC submission on ***_trollme for some laughs

### PR DESCRIPTION
**Pull request by PangJJ**
@MrMarciaOng @kohyida1997 wreck my PR. Too long since I've last touched JS

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -344,12 +360,17 @@ bot.on('message', async (msg) => {
   if (msg.photo && msg.caption) {
     const match = msg.caption.match(/#LC(20\d{2})(\d{2})(\d{2})/g);
     const matchTT = msg.caption.match(/#LCTT(20\d{2})(\d{2})(\d{2})/g); // #LCTT (time travel) for submission of past LCs. Note that this will accept any date
+    const matchTroll = msg.caption.match(/#LC(20\d{2})(\d{2})(\d{2})_trollme/g);
```
Comment (kohyida1997) [inline index.js:363]: `troll` is only true iff `(match && matchTroll)`. We could simplify the code to not use a separate `troll` variable. 

Consider:
```
const useTrollQuote = msg.caption.match(/#LC(20\d{2})(\d{2})(\d{2})_trollme/g) 
    && match
    && (trollQuotes.length > 0);
```
and then later...
```
let quote = !useTrollQuote 
    ? response.data
    : trollQuotes[Math.floor(Math.random() * array.length)];
}
```

Comment (PangJJ) [reply index.js]: fixed

File: index.js [hunk 2]
```diff
@@ -409,7 +430,11 @@ bot.on('message', async (msg) => {
 
       console.log('dateStr', dateStr);
       console.log('statsStr', statsStr);
-      reply = `Good job doing ${dateStr} LC question! 🚀 ${namePart}${statsStr}\r\n${response.data}`;
+
+      const trollQuoteChoice = Math.floor(Math.random() * array.length);
```
Comment (kohyida1997) [inline index.js:434]: Oops I think it should be

`const trollQuoteChoice = Math.floor(Math.random() * trollQuotes.length);`

Comment (PangJJ) [reply index.js]: RAWR i shouldn't write stuff past midnight

**Comment by jymchng** [comment]
Leaving a comment here for momento

**Review by MrMarciaOng — APPROVED** [review]

**Pull request merged**